### DISCUSSION TITLE
Unable to change content of textbox by name

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#2162] [Q&A] [answered]
Hi I am trying to change the content field or the text to be displayed in the textbox by name but i am not able to do so. 

```
q.page['most_graph'] = ui.form_card(
            box=ui.box(zone='top_g', order=1, width='50%',height='410px'),
            title='',
            items=[
                ui.inline(
                            items=[
                            ui.text_m(content="Overall",name='tog_pop_txt'),
                            ui.menu(items=[
                            ui.command(name='tog_pop_o', label='Overall',value='pop_o',), 
                            ui.command(name='tog_pop_3', label='Last 3 Months',value='pop_3'), 
                            ui.command(name='tog_pop_today', label='Today', value='pop_today') 
                        ] )]
                            align='center',justify='end'
                        )
            ],
        )
```

I just need to update the value of the label to the selected `ui.command` value. This is how I'm updating the value:
`q['most_graph'].tog_pop_txt.content="Last 3 Months"`

**Comment by mturoci** [comment]
Hi @n-srinidhi!

Please provide a minimal example that I can run myself and tell you whether the problem is on our side or yours. Thanks!

**Comment by mtanco** [comment, marked as answer]
HI @n-srinidhi !

The code you pasted, `q['most_graph'].tog_pop_txt.content="Last 3 Months"`, has a typo in that we want `q.page[]...`. When I changed to that the text updated. Hope it helps.
